scripts/ij.bazelproject

```diff
@@ -1,14 +1,14 @@
 # Setup IntelliJ for Bazel development using the IntelliJ Bazel Plugin.
 # See https://github.com/bazelbuild/intellij for installation instructions.
 directories:
-  src/main
-  src/test
+  .
```

Comment:
Yes, but that doesn't seem to cause any issues for the IDE. I've been running with this project file for some time now and macOS and Linux.

Alternatively we could also explicitly list all directories, but that would prevent us from editing files in the root directory using the IDE.